clang/lib/Driver/ToolChains/MSVC.cpp

```diff
@@ -322,6 +322,9 @@ void visualstudio::Linker::constructMSVCLibCommand(Compilation &C,
     }
     CmdArgs.push_back(II.getFilename());
   }
+  if (Args.hasArg(options::OPT_fsycl_link_EQ) &&
+      Args.hasArg(options::OPT_fintelfpga))
+    CmdArgs.push_back(C.getArgs().MakeArgString(Twine("/IGNORE:4221")));
```

Comment:
```suggestion
    CmdArgs.push_back("/IGNORE:4221");
```

Reply:
Additional allocation not needed with string literal.

Reply:
Fixed